chapters/connectors.tex

```diff
@@ -384,7 +384,7 @@ \section{Generation of Connection Equations}\label{generation-of-connection-equa
 \begin{lstlisting}[language=modelica]
 connect(a, b);
 \end{lstlisting}
-the primitive components of \lstinline!a! and \lstinline!b! form a connection set, together with an indication of whether they are from an inside or an outside connector.
+one connection set is generated for every pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

Comment:
Tone down emphasis on the count:
```suggestion
a connection set is generated for every pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

Reply:
I'm not sure which variant is best; I can understand that being clear that it is one for each pair might avoid ambiguities.

Reply:
Maybe _a… each_ is better?
```suggestion
a connection set is generated for each pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```